src/Microsoft.AspNetCore.Razor.Language/DefaultRazorParserOptionsBuilder.cs

```diff
@@ -19,9 +20,11 @@ public DefaultRazorParserOptionsBuilder(bool designTime)
 
         public override bool ParseLeadingDirectives { get; set; }
 
+        public override RazorParserVersion Version { get; set; }
```

Comment:
We should treat `Version` like we do `DesignTime`, aka required at construction of the builder.